index.html

```diff
@@ -47,6 +45,186 @@ <h5>LET'S CONNECT</h5>
+              <li><div id="work4counter1"></div></li>
+              <li><div id="work4text"><h5>2015</h5></div></li>
+            </ul>
+          </div>
+          <div class="work4primary-text">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+          </div>
+          <div id="work4tags">
+            <ul>
+              <li><div id="work4tag1"><div id="html4">html</div></div></li>
+              <li><div id="work4tag2"><div id="css4">css</div></div></li>
+              <li><div id="work4tag3"><div id="javascript4">javascript</div></div></li>
+            </ul>
+          </div>
+          <div class="work4Actions">
+              <button type="button" id="work4see-project">
+                <strong>See Project</strong>
+              </button>
+          </div>
+        </div>
+      </section>
+    </section>
+    <section class="about-myself">
+      <div class="about-myself-primary-text">
+        <h2>About Myself</h2>
+      </div>
+      <div class="about-myself-primary-bodytext">
+        <p>Hello I'm a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+      </div>
+      <div class="about-myself-social-media-text">LET'S CONNECT</div>
+      <div id="about-myself-social-media-icons">
+        <ul>
+          <li><a href="#"><img src="./images/Vector2.svg" alt="GitHub"></a></li>
+          <li><a href="#"><img src="./images/Linkedinicon.svg" alt="Linkedin"></a></li>
+          <li><a href="#"><img src="./images/Vector3.svg" alt="Peace"></a></li>
+          <li><a href="#"><img src="./images/Vector.svg" alt="Twitter"></a></li>
+          <li><a href="#"><img src="./images/Vector1.svg" alt="Mail"></a></li>
+        </ul>
+      </div>
+      <button type="button" id="my-resume"><span>Get my resume</span></button>
+      <section class="skills">
+        <div class="skill-list">
+          <div id="ul-languages">
+            <h5>Languages</h5><span id="open-chevron"><img src="./images/Icon-Chevron-Down-Larger.png" alt="Open"></span>
+            <ul id="lang-list">
+              <li id="javascript">
+                <div id="img"><img src="./images/javascript.png" alt="Javascript"></div>
+                <div>Javascript</div>
+              </li>
+              <li id="html">
+                <div id="img1"><img src="./images/html.png" alt="HTML"></div>
+                <div>HTML</div>
+              </li>
+              <li id="css">
+                <div id="img2"><img src="./images/css.png" alt="CSS"></div>
+                <div>CSS</div>
+              </li>
+            </ul>
+          </div>
+          <div id="ul-frameworks">
+            <h5>Frameworks</h5><span id="upper-chevron"><img src="./images/Disabled.png" alt="Disabled"></span>
```

Comment:
Kindly use a more relatable value for the `alt` attribute. This is what the user gets to see when there is an issue displaying the image.